apps/vessel-history/features/map/popups/PopupWrapper.tsx

```diff
@@ -0,0 +1,57 @@
+import { useCallback } from 'react'
+import cx from 'classnames'
+import { Popup } from 'react-map-gl'
+import { ExtendedFeature } from '@globalfishingwatch/react-hooks'
+import { Icon } from '@globalfishingwatch/ui-components'
+import { dataviewInstances } from 'features/dataviews/dataviews.config'
+import styles from './Popup.module.css'
+
+type PopupWrapperProps = {
+  layers: ExtendedFeature[]
+  className?: string
+  latitude: number
+  longitude: number
+}
+function PopupWrapper({ layers, className = '', latitude = 0, longitude = 0 }: PopupWrapperProps) {
+  const getLayerColor = useCallback(
+    (datasetId: string) => {
+      if (datasetId.includes('public-eez')) {
+        return dataviewInstances.find((layerConfig) => layerConfig.id === 'public-eez-areas')
+          ?.config.color
+      }
```

Comment:
It would be cleaner if we don't have special cases, would be possible to rename the `dataviewInstance.id` to `public-eez-land` so that it matches the `datasetId` to get rid of this special case?